hack/kops-patch.yaml

```diff
@@ -8,8 +8,14 @@ spec:
             "elasticfilesystem:CreateAccessPoint",
             "elasticfilesystem:DeleteAccessPoint",
             "elasticfilesystem:DescribeFileSystems",
-            "elasticfilesystem:DescribeAccessPoints"
+            "elasticfilesystem:DescribeAccessPoints",
+            "elasticfilesystem:DescribeMountTargets",
+            "ec2:DescribeAvailabilityZones"
           ],
           "Resource": "*"
         }
       ]
+
+  cloudConfig:
+    awsEBSCSIDriver:
```

Comment:
`awsEFSCSIDriver`?

Reply:
When k8s is installed through kops, ebs driver enables as a default since k8s 1.22

Reply:
when you say `This config managed ebs driver`, could you forward the link of the resource? and what do you mean by `ebs driver enables as a default`?

Reply:
[New clusters running Kubernetes 1.22 will have AWS EBS CSI driver enabled by default.
](https://github.com/kubernetes/kops/blob/master/docs/releases/1.22-NOTES.md#:~:text=New%20clusters%20running%20Kubernetes%201.22%20will%20have%20AWS%20EBS%20CSI%20driver%20enabled%20by%20default.)

Reply:
I guess what I am confused here is I do not fully understand how is this controlled and why do we need to control it through ebs driver?

Could you add more details/explanations here? thanks

Reply:
@Ashley-wenyizha To clarify, this patch simply disables the installation of the EBS CSI Driver during cluster creation with kOps. This is not being through the EBS CSI Driver -- it is part of kOps cluster configuration.

Reply:
yep I have communicated offline with Soyeon. lgtm